### PR TITLE
PMM-4763

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**
Basic questions answered.

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
Reviewers: Please don't just accept/reject. Please also discuss, suggest.

Motivation: To move some FAQ items to more natural homes, but also add some basic questions which have no natural home, for SEO and first-timers.

File: docs/faq.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+
+- Track performance over time
+- Identify problem queries
+- Identify latencies and resource bottlenecks
+
+## How much does it cost?
+
+PMM is free if you run PMM on your own hardware.
+
+If you run on a cloud service (Amazon AWS) there are infrastructure fees.
```
Comment (shoffman-percona) [inline docs/faq.md:19]: there may be fees...only if you deploy from the marketplace...  your ec2 instances with the pmm binaries are still free

File: docs/faq.md [hunk 2]
```diff
@@ -19,6 +31,21 @@
     - [Setting up PMM Server](setting-up/server/index.md)
     - [Setting up PMM Client](setting-up/client/index.md)
```
Comment (shoffman-percona) [inline docs/faq.md:32]: does this go right to the "requirements" or just the page and the user has to figure it out (can't easily follow from git)...Can we recap the basic requirements here and use these links for more details?

File: docs/faq.md [hunk 3]
```diff
@@ -19,6 +31,21 @@
...
+## How can I contact the developers?
+
+- [Community forum](https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management).
+- [Discord chat](http://per.co.na/discord).
+- [PMM project in JIRA](https://jira.percona.com/projects/PMM).
+
+
 ## How can I upgrade from version 1?
 
 There is no direct software upgrade path.
```
Comment (shoffman-percona) [inline docs/faq.md:51]: somewhere I found a link with more detail in the docs that talks about the architectural changes...

Here: can we link here for more info: how-to/upgrade.html#upgrade-from-pmm-1  (not sure how to make the anchor link with the .md extension...)

**Review by shoffman-percona — COMMENTED** [review]
My thoughts

**Pull request closed without merging**